src/constants.js

```diff
@@ -1564,21 +1564,31 @@ export const GLSL1 = '100';
 export const GLSL3 = '300 es';
 
 /**
- * WebGL coordinate system.
+ * WebGL coordinate system [-1, 1].
  *
  * @type {number}
  * @constant
  */
 export const WebGLCoordinateSystem = 2000;
 
 /**
- * WebGPU coordinate system.
+ * WebGPU coordinate system [0, 1].
  *
  * @type {number}
  * @constant
  */
 export const WebGPUCoordinateSystem = 2001;
 
+/**
+ * Reversed coordinate system [1, 0].
+ *
+ * Used for reverse depth buffer.
+ *
+ * @type {number}
+ * @constant
+ */
+export const ReversedCoordinateSystem = 2002;
```

Comment:
Do we not technically need "ReversedWebGPUCoordinateSystem" and "ReversedWebGLCoordinateSystem"? Or is this just for WebGPU?

Reply:
At some point reverse depth buffer will also be supported in `WebGPURenderer`. If computations in `Matrix4` differ in WebGL and WebGPU, then we should indeed rename to `ReversedWebGLCoordinateSystem`.

Reply:
It is implemented exactly the same in WebGL and WebGPU as reverse depth needs a [0, 1] NDC range which all modern graphics APIs use. It is only OpenGL which uses [-1, 1] with precision stuck uselessly in the middle. I explain this better in previous comments. The only other constant I would consider would be for an infinite-reversrd matrix which only works for perspective.